src/node_file.cc

```diff
@@ -1085,7 +1085,7 @@ v8::CFunction fast_internal_module_stat_(
     v8::CFunction::Make(FastInternalModuleStat));
 
 constexpr bool is_uv_error_except_no_entry(int result) {
-  return result < 0 && result != UV_ENOENT;
+  return result < 0 && !(result & (UV_ENOENT | UV_ENOTDIR));
```

Comment:
libuv errors do not support bitwise operations, so this definitely doesn't work

```suggestion
  return result < 0 && !(result == UV_ENOENT || result == UV_ENOTDIR);
```

Reply:
So true, I've fixed that! Thanks!